### ISSUE TITLE
Headings can be accidentally nested within themselves

### ISSUE THREAD

**Issue by jaredreich** [jaredreich/pell#93]
Repro steps:
- Create a list (orderded or unordered)
- Click a header button multiple times
- See (in the output HTML) that the header nest inside itself each time

**Comment by Sjeiti** [comment]
While html should follow basic W3C rules, a quick (not-really-a) fix would be a button that removes the formatting from the current selection.

**Comment by jaredreich** [comment]
@Sjeiti see "removeFormat" https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand

**Comment by Sjeiti** [comment]
@jaredreich yeah I tried that with Pell. It removes formatting set by style but not formatting implied by element. I'm afraid you'll need a custom method to strip all elements for that (also needed for #53).
A really easy `content.textContent += ''` works but also strips the linebreaks. So you might need something fancier.

**Comment by jaredreich** [comment]
@Sjeiti ok I see, thanks for the investigation. I'll have to dig deeper.

**Comment by Mera-Gangapersaud** [comment]
I found [this](https://stackoverflow.com/questions/22725956/formatblock-command-does-not-overwrite-existing-block-on-list-item) on StackOverflow to help explain the issue. Looks like it could be fixed from chromium since the problem is with how the browser determines the blocks used by formatBlock.
